src/Fieldset/README.md

```diff
@@ -0,0 +1,41 @@
+# Fieldset
+
+Wraps many inputs in a border and allows validating them as one form control.
+
+## API
+
+### `children` (node; optional)
+`children` specifies the elements that the Fieldset will wrap. The default is null.
+
+### `className` (string array; optional)
+`className` specifies Bootstrap class names to apply to the input component. The default is an empty array.
+
+### `id` (string; optional)
+`id` specifies a unique identifier for the fieldset.  It will be used to create an id for the ValidationMessage that is referenced by an `aria-describedby` on the `fieldset`. The default is an auto-generated id of the form `fieldsetN` where N is an auto-incremented integer.
+
+### `invalidMessage` (string or element; optional)
+`invalidMessage` specifies the message to display immediately below the fieldset outside the border when `isValid` is false. The default is an empty string.
+
+### `isValid` (boolean; optional)
+`isValid` specifies whether the whole fieldset has validated correctly. Consider updating this from an `onSubmit` handler of a parent `form` element. The default is true.
+
+### `legend` (string or element; required)
+`legend` specifies the label to be used for the overarching fieldset. This can be a string or element type. It appears at the top of the fieldset, inline with the border.
+
+### `variant` (object with a string `status` field; optional)
+`variants` specifies which type of fieldset to display. These are the Variants currently supported:
+
+#### `status`
+
+**INFO** (default):
+* Fieldset border is always black, even when `isValid` is `false`.
+* Displays `invalidMessage` in black.
+* Does not display an icon next to `invalidMessage`.
+
+**DANGER**:
+* Fieldset border is red when `isValid` is `false` and black when `true`.
+* Displays `invalidMessage` in red.
+* Displays an "exclamation" icon next to `invalidMessage`.
+
+### `variantIconDescription` (string or element; optional)
```

Comment:
If `variant` is going to be an object, wouldn't it make sense for it's icon description to live within that object?

Reply:
I think `variant` is intended just to specify one theme out of an enum. If we start putting display strings in it then I'm afraid we'll start dumping any theme-related data into `variant`.